examples/libscrypt/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(libscrypt-test)
+
+hunter_add_package(libscrypt)
```

Comment:
Plus `DOCUMENTATION_START` / `DOCUMENTATION_END` for `literalinclude`, see:
* https://github.com/ruslo/hunter/blob/b9ec25a0ffd23eeccdb692edef47f1cd03db30f9/examples/foo/CMakeLists.txt#L12-L18